<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FieldDialog</class>
 <widget class="QDialog" name="FieldDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>212</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="group_box">
     <property name="title">
      <string>[Field Properties]</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="name_label">
        <property name="text">
         <string>[&amp;Name:]</string>
        </property>
        <property name="buddy">
         <cstring>name_edit</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="name_edit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="type_label">
        <property name="text">
         <string>[&amp;Type:]</string>
        </property>
        <property name="buddy">
         <cstring>type_edit</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="type_edit">
        <property name="editable">
         <bool>true</bool>
        </property>
        <item>
         <property name="text">
          <string>INTEGER</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>TEXT</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>REAL</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>BLOB</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>NULL</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="dflt_label">
        <property name="text">
         <string>[&amp;Default value:]</string>
        </property>
        <property name="buddy">
         <cstring>dflt_edit</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="dflt_edit"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="null_label">
        <property name="text">
         <string>[NOT N&amp;ULL flag:</string>
        </property>
        <property name="buddy">
         <cstring>null_box</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QCheckBox" name="null_box">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="primary_key_label">
        <property name="text">
         <string>[&amp;Primary key:]</string>
        </property>
        <property name="buddy">
         <cstring>pk_box</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QCheckBox" name="pk_box">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>7</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancel_button">
       <property name="text">
        <string>[&amp;Cancel]</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ok_button">
       <property name="text">
        <string>[&amp;OK]</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
